### PR TITLE
add update user preferences

### PR DESCRIPTION
**Pull request by jluribe**
your incredible giphy

### What?
- Add services to integration, skore knowledge API and authentications
- Add middleware service to send excepcions to Sentry services.

### Why?
cause, the chatbot have to be enable to generate session token and query contents by current user login to Meya through Smooch middleware.

### CODE REVIEW

File: skore/integration_service.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import requests
+
+MEYA_ERROR = "{host}/integration/v1/meya/errors"
+
+class IntegrationService:
```
Comment (nicolaslima) [inline skore/integration_service.py]: @jluribe I have two doubts
What is a `service` inside the SDK? And what is the purpose of this service?
I think this name is very generic from what I understand

File: skore/meya/user.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+class User(object):
+  def __init__(self, id):
+    self.id = id
+    self.jwt_token = None
+    self.refresh_token = None
+    self.skore_user_id = None
+    self.meya_user = None
+
+  def signed_in(self):
```
Comment (nicolaslima) [inline skore/meya/user.py]: @jluribe as this is a boolean method, maybe prefix with `is`, what you think? `def is_signed_in(self):`

File: skore/meya/user.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      return self.jwt_token != None and self.refresh_token != None and self.skore_user_id != None
+
+  def clean_up_credentials(self, sync = True):
+    self.jwt_token = None
+    self.refresh_token = None
+    if sync: self.__sync()
+
+  def set_skore_user_id(self, skore_user_id, sync = True):
+    pass
```
Comment (nicolaslima) [inline skore/meya/user.py]: @jluribe I did not understand, could you explain? :D

File: skore/meya/user.py (hunk 2 above)
Comment (jluribe) [reply skore/meya/user.py]: @nicolaslima make a sense, it was rename.

File: skore/meya/user.py (hunk 3 above)
Comment (jluribe) [reply skore/meya/user.py]: @nicolaslima It is an abstract method.

File: skore/integration_service.py (hunk 1 above)
Comment (jluribe) [reply skore/integration_service.py]: @nicolaslima class moved to https://github.com/skore-io/skore-chatbot-sdk/blob/feature/update_user_preferences_chatbot_intro/skore/meya/issue_tracker.py#L74

**Pull request merged**